gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-api/src/main/pegasus/gobblin/service/FlowConfig.pdsc

```diff
@@ -0,0 +1,38 @@
+{
+  "type" : "record",
+  "name" : "FlowConfig",
+  "namespace": "gobblin.service",
+  "doc" : "Configuration for a flow",
+  "fields" : [
+    {
+      "name" : "flowName",
+      "type" : "string",
+      "doc" : "Name of the flow"
+    },
+    {
+      "name" : "flowGroup",
+      "type" : "string",
+      "doc" : "Group of the flow. This defines the namespace for the flow."
+    },
+    {
+      "name" : "schedule",
+      "type" : "string",
+      "doc" : "Schedule for flow in cron format"
```

Comment:
Think about choosing a typeref for validation or using a validator directly here.